src/dpp/channel.cpp

```diff
@@ -478,14 +478,34 @@ channel& channel::fill_from_json(json* j) {
 }
 
 std::string thread::build_json(bool with_id) const {
-	json j = json::parse(channel::build_json(with_id));
```

Comment:
removing this line removes all attributes from channel we might want to keep like type, name, parent id, etc?

Reply:
Yeah, I completely forgot about the use case of creating threads. I'll see if we could maybe keep the old info whilst updating stuff.